app/javascript/src/Policies/components/CustomPolicy.jsx

```diff
@@ -46,17 +46,30 @@ function CSRFToken ({win = window}: {win?: any}) {
 
 function CustomPolicyForm ({policy, onChange}: {policy: Policy, onChange: OnChange}) {
   const isNewPolicy = !(policy.id && policy.id !== 0)
-  const action = (isNewPolicy) ? '/p/admin/registry/policies/' : `/p/admin/registry/policies/${policy.id}`
+  const initialState = () => {
+    return (isNewPolicy)
+      ? { action: '/p/admin/registry/policies/', method: 'post', submitText: 'Create Custom Policy' }
+      : { action: `/p/admin/registry/policies/${policy.id}`, method: 'put', submitText: 'Update Policy Schema' }
+  }
+  const [state, setState] = useState(initialState())
```

Comment:
originally I did it that way, since method is the only thing that changes. Kind of liked the idea of having the whole form modifiers in the state, but it's clearly not worthy. That's why the initialState fn above.